source/__snapshots__/lib.test.ts.snap

```diff
@@ -1,117 +1,147 @@
 // Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
 exports[`init - registerContentScript > should register multiple manifest scripts on new permissions 1`] = `
-[MockFunction registerContentScript] {
+[MockFunction spy] {
   "calls": [
     [
-      {
-        "allFrames": undefined,
-        "css": [],
-        "excludeMatches": undefined,
-        "js": [
-          "/script.js",
-        ],
-        "matches": [
-          "https://granted.example.com/*",
-        ],
-        "runAt": undefined,
-      },
+      [
+        {
+          "allFrames": undefined,
+          "css": undefined,
+          "excludeMatches": [
+            "https://content-script.example.com/*",
```

Comment:
Upgrade. However not really useful because Chrome natively deduplicates scripts now.